src/test/java/com/basiscomponents/db/util/ResultSetProvider.java

```diff
@@ -4,10 +4,48 @@
 
 public class ResultSetProvider {
 
-  public static ResultSet createDefaultResultSet() throws Exception {
+  public static ResultSet createDefaultResultSet(boolean nullAllFields) throws Exception {
```

Comment:
Hier beim Nächsten Mal aufpassen, die Methode wurde woanders genutzt und somit hatte eine andere Klasse einen Compile Error. 
```suggestion

  public static ResultSet createDefaultResultSet() throws Exception {
         return createDefaultResultSet(false); 
  }
  public static ResultSet createDefaultResultSet(boolean nullAllFields) throws Exception {
```
Somit wäre die originale Methodensignatur weiterhin vorhanden